src/config/android/findAndroidAppFolder.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs');
+const path = require('path');
+
+module.exports = function findAndroidAppFolder(folder) {
```

Comment:
Shouldn't we revert the comment here? + maybe include some `@todo` with our planned rewrite